### DISCUSSION TITLE
Using links in the meta data absolute or relative?

### DISCUSSION THREAD

**Discussion by christoph-LSN** [open-sdg/open-sdg#1520] [Q&A] [answered]
@brockfanning @MoBosse 

For each indicator I have an excel-sheet which can be downladed.

The link is saved in my metadata files like here:

source_url_1: 'https://christoph-lsn.github.io/MT_Site/assets/excel/2021_1-1-1.xlsx'

https://github.com/christoph-LSN/IM_Data/blob/56423608ebee80999caefe8627b3a16d66e131d3/meta/1-1-1.md?plain=1#L29

In the end my Migration Report will be published on https://integrationsmonitoring.niedersachsen.de/ . In general I think it will work using the link to the excel file in my site repo. 

Two thoughts on this:
1) Can I use relative links like ../MT_Site/assets/excel/2021_1-1-1.xlsx
2) And sometimes I link two or more indicators in the indicator description, How this link has to look like?  https://integrationsmonitoring.niedersachsen.de/1-1-1
or
https://christoph-lsn.github.io/MT_Site/1-1-1/
(To my mind I can use https://christoph-lsn.github.io/MT_Site/1-1-1/ because of the DNS value the site will be redirected anyway... haven't tested it yet)

What do you think?

[data repo](https://github.com/christoph-LSN/MT_Site)
[site repo](https://github.com/christoph-LSN/IM_Data)

[site](https://christoph-lsn.github.io/MT_Site/)

**Comment by brockfanning** [comment]
@christoph-LSN Regarding local links in those metadata "_url" fields -- looking at the code, it seems like if the metadata value does not contain "http" or "mailto" that it will not even display the link. So, using `source_url_1: ../assets/excel/my-file.xlsx` would not work (the link would not display at all).

One option that came to my mind is to have Open SDG assume that if it does not contain "http" or "mailto" that it must be a local path. I put up an experimental PR if you would like to test that out: #1794. In theory this would allow you to use something like: `source_url_1: assets/excel/my-files.xlsx`, and it should work for both staging and production.

**Comment by christoph-LSN** [comment, marked as answer]
@brockfanning @MoBosse 

Thank you Brock! it works as you can see here: https://mt-production.github.io/1-1-1/
